spec/keyboard-commands.spec.js

```diff
@@ -54,13 +54,36 @@ describe('KeyboardCommands TestCase', function () {
                             command: 'superscript',
                             key: 'p',
                             meta: true,
-                            shift: false
+                            shift: false,
+                            alt: false
                         },
                         {
                             command: 'subscript',
                             key: 'p',
                             meta: true,
-                            shift: true
+                            shift: true,
+                            alt: false
+                        },
+                        {
+                            command: 'h1',
```

Comment:
The command you want is `append-h1`, `append-h2`, `append-h3`, etc.  These are the built-in actions that are triggered when you click the various buttons.